src/views/WellnessCheck/components/Denied/Denied.js

```diff
@@ -0,0 +1,119 @@
+import Grid from '@material-ui/core/Grid';
```

Comment:
Same comments as Approved.js.  They are so similar, could we have a "HealthStatus" component instead, with only a few differences?

Reply:
merged the separate components into one file called "healthStatus" working and ready for review